eox_hooks/tests/test_receivers.py

```diff
@@ -0,0 +1,55 @@
+"""This file contains all the test for the receivers.py file.
+
+Classes:
+    SingalsTestCase: Test Receivers.
+"""
+from django.conf import settings
+from django.test import TestCase, override_settings
+from mock import Mock, patch
+
+from eox_hooks.receivers import hooks_handler
+
+
+class TestReceivers(TestCase):
+    """Receivers test class."""
+
+    def setUp(self):
+        """Set up class for Receivers testing."""
+        self.sender = Mock()
+        self.trigger_event = "sender.example"
+
+    @patch("eox_hooks.receivers.action_handler")
+    def test_trigger_response_action(self, action_handler):
+        """
+        Used to test that if USE_EOX_HOOKS is set to true then the handler triggers
+        the execution of the action handlers.
+        """
+        configuration = settings.EOX_HOOKS_DEFINITIONS.get(self.trigger_event)
+
+        hooks_handler(self.sender, trigger_event=self.trigger_event)
```

Comment:
you have changed the method, this should be with signal instead of trigger event